### ISSUE TITLE
Training stock at 0% at first epoch

### ISSUE THREAD

**Issue by aoussou** [PyTorchLightning/pytorch-lightning#6512]
Hello,

Training stock at 0% at the very first epoch whether using `fast_dev_run` or not. Not error reported. 

How can I debug that?

> Using poly LR scheduler with warm-up epochs of 0!

>   | Name      | Type               | Params
> -------------------------------------------------
> 0 | model     | EncNet             | 36.0 M
> 1 | criterion | SegmentationLosses | 0     
> -------------------------------------------------
> 36.0 M    Trainable params
> 0         Non-trainable params
> 36.0 M    Total params
> 144.163   Total estimated model params size (MB)
> Epoch 0:   0%|                                                                                                                                                                        | 0/1 [00:00<?, ?it/s]
>

**Comment by awaelchli** [comment]
I had the same on a machine with the 3090 GPU.
Updating torch to the newest version solves this problem. Make sure to have cudatoolkit=11.0, i.e.: `conda install pytorch cudatoolkit=11.0 -c pytorch`

**Comment by aoussou** [comment]
@awaelchli the code I'm using can only run with Cuda 10.2, does this mean Lightning is unusable in this case?

**Comment by aoussou** [comment]
@awaelchli 

Thank you for the explanation. Does this mean that lightning is not fully compatible with older versions of cuda/pytorch?

Is there a reference document that outlines compatibility requirements?

**Comment by awaelchli** [comment]
no but the 3090 series gpus require a newer pytorch version to function optimally. All I'm saying is if you have one (I don't know) then updating will solve the problem.

**Comment by aoussou** [comment]
@awaelchli Thank you. I have 2 x 2080 Ti .

I updated as you suggested but unfortunately it didn't fix it:

cudatoolkit               11.0.221             h6bb024c_0
python                    3.8.8                hdb3f193_4  
pytorch                   1.7.1           py3.8_cuda11.0.221_cudnn8.0.5_0    pytorch
pytorch-lightning         1.2.3              pyhd8ed1ab_0    conda-forge
torchvision               0.8.2                py38_cu110    pytorch

You labelled the issue with "information needed", please let me know what information I can supply.

Thank you for your help, I appreciate.

**Comment by awaelchli** [comment]
Still a problem? Can you confirm this is not just due to stalling of dataloader workers spawning context? Set `num_workers=0` as a test.

**Comment by aoussou** [comment]
@awaelchli 

Sorry, I didn't get it to work and I'm afraid I don't have time to investigate this further. You may close this issue if you wish.

**Comment by awaelchli** [comment]
Okay, anyway thanks for your patience. If this occurs again reach out to me here or in our [slack](https://join.slack.com/t/pytorch-lightning/shared_invite/zt-f6bl2l0l-JYMK3tbAgAmGRrlNr00f1A).
